src/types/output_damage.zig

```diff
@@ -0,0 +1,52 @@
+const wlr = @import("../wlroots.zig");
+
+const pixman = @import("pixman");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+
+pub const OutputDamage = extern struct {
+    output: *wlr.Output,
+    max_rects: i32,
```

Comment:
```suggestion
    max_rects: c_int,
```

Reply:
When should `c_int` be used instead of `i32`?
E.g `Output.width` is `i32` and I don't yet see how this is different.

Reply:
> When should `c_int` be used instead of `i32`?
> E.g `Output.width` is `i32` and I don't yet see how this is different.

`wlr_output.width` is a `int32_t` which is guaranteed to be 32 bits on all targets. Hovewer, `wlr_output_damage.max_rects` is a `int` which is only defined to be at least 16 bits by the standard. Zig's `c_int` matches whatever size C's `int` has for a given target.

Reply:
Thanks!